packages/token-lab/src/hooks/tokens.ts

```diff
@@ -111,6 +122,22 @@ export async function loadTokens(filename: string): Promise<string> {
  * Save tokens to IndexedDB (NOT Jotai!)
  */
 export async function saveTokens(filename: string, tokens: string): Promise<void> {
+  // TODO: this logic should instead be coming from <App onUpdate={newTokens => { ... }} />
```

Comment:
to implement the api documented when token lab was called terrazzo gallery, im guessing we will have to store the onUpdate function in a context provider. is there a cleaner approach?